diff --git a/addtocal.module b/addtocal.module
index 333ef5f..3c51735 100644
--- a/addtocal.module
+++ b/addtocal.module
@@ -89,16 +89,12 @@ function addtocal_get_addtocal_entities() {
  * Implements hook_field_formatter_info().
  */
 function addtocal_field_formatter_info() {
+  $date_info = date_field_formatter_info();
   return array(
     'addtocal_view' => array(
       'label' => t('Add to Cal'),
       'field types' => array('date', 'datestamp', 'datetime'),
       'settings' => array(
-        'format_type' => 'long',
-        'multiple_number' => '',
-        'multiple_from' => '',
-        'multiple_to' => '',
-        'fromto' => 'both',
         'location' => array(
           'field' => -1,
           'tokenized' => '',
@@ -111,7 +107,7 @@ function addtocal_field_formatter_info() {
         'past_events' => FALSE,
         'show_repeat_rule' => 'hide',
         'view_mode' => '',
-      ),
+      ) + $date_info['date_default']['settings'],
     ),
   );
 }